change/@ni-nimble-components-44c2b207-def9-472c-ae87-f6e48877185e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
You could consider mentioning this PR (or summarizing the proposal) on the [UX Design Discussion](https://teams.microsoft.com/l/channel/19%3A8786e8b04c62455798795d0a9d7aa362%40thread.tacv2/UX%20Design%20Discussion?groupId=9ee065d7-3898-4245-82f6-76e86084b8b1&tenantId=87ba1f9a-44cd-43a6-b008-6fdb45a5204e) channel. It's possible IxDs will have other ideas but even if they don't it'd be great to start engaging them.